modules/ossm-about-gateway-injection.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+
+// gateways/ossm-about-gateways.adoc
+
+:_mod-docs-content-type: Concept
+[id="ossm-about-gateway-injection_{context}"]
+= About gateway injection
+:context: ossm-about-gateway-injection
+
+Gateway injection relies upon the same mechanism as sidecar injection to inject the Envoy proxy into gateway pods. To install a gateway using gateway injection, you create a Kubernetes `Deployment` and an associated Kubernetes `Service` in a namespace that is visible to the {istio} control plane. When creating the `Deployment` you label and annotate it so that the {istio} control plane injects a proxy, and the proxy is configured as a gateway. After installing the gateway, you configure it to control ingress and egress traffic using the {istio} `Gateway` and `VirtualService` resources.
```

Comment:
Per [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references), object references take the work _object_. 
```suggestion
Gateway injection relies upon the same mechanism as sidecar injection to inject the Envoy proxy into gateway pods. To install a gateway using gateway injection, you create a Kubernetes `Deployment` object and an associated Kubernetes `Service` object in a namespace that is visible to the {istio} control plane. When creating the `Deployment` object you label and annotate it so that the {istio} control plane injects a proxy, and the proxy is configured as a gateway. After installing the gateway, you configure it to control ingress and egress traffic using the {istio} `Gateway` and `VirtualService` resources.
```

Reply:
done